webservices/resources/elections.py

```diff
@@ -280,6 +288,7 @@ def join_candidate_totals(query, kwargs, totals_model):
         sa.and_(
             CandidateHistory.candidate_id == CandidateCommitteeLink.candidate_id,
             CandidateHistory.two_year_period == CandidateCommitteeLink.fec_election_year,
+            CandidateCommitteeLink.committee_designation.in_(['P', 'A']),
```

Comment:
A comment on why this is needed may be useful. Also, should we create a test that would add other committee designations and assert that they are *not* picked up?

Reply:
Here is a comment on why this is needed:
Joint Fundraising Committees raise money for multiple candidate committees and transfer that money to those committees. By limiting the committee designations to A and P you eliminate J (joint) and thus do not inflate the candidate's money by including it twice and by including money that was raised and transferred to the other committees in the joint fundraiser.